test/uk/gov/hmrc/traderservices/controllers/VesselDetailsFormSpec.scala

```diff
@@ -117,7 +118,7 @@ class VesselDetailsFormSpec extends UnitSpec with FormMatchers {
         .updated("timeOfArrival.minutes", "")
       form.bind(input).value shouldBe None
       form.bind(input).errors should haveOnlyErrors(
-        FormError("timeOfArrival", "error.timeOfArrival.all.required")
+        FormError("timeOfArrival", Seq("subfieldFocus=hour", "error.timeOfArrival.all.required"))
```

Comment:
Time of arrival is fully missing in this test

Reply:
???